### PR TITLE
Trebmuh fixes

### PR DESCRIPTION
**Pull request by trebmuh**
Hi @peterdd.

Here are a bunch of fixes/improvements to the current manual that I've been reading entirely today.
I hope this helps somewhat to improve FlySpray.

Olivier

### CODE REVIEW

**Comment by peterdd** [comment]
Please do not put too much into one PR. One obstacle/problem and it delays the possible good stuff to merge.

File: _manual/categories_and_tags.md [hunk 1]
```diff
@@ -10,33 +10,41 @@ With Flyspray 1.0 tags are supported. They are just labels that can be applied t
 
 The differences between **categories** and **tags** in Flyspray 1.0 are:
 
-##Categories
+Categories
```
Comment (peterdd) [inline _manual/categories_and_tags.md]: why not just fix  by adding the space for `## Categories`?

Comment (trebmuh) [reply _manual/categories_and_tags.md]: I didn't think it was working too. Thanks.
PR modified.

**Comment by trebmuh** [comment]
> Please do not put too much into one PR. One obstacle/problem and it delays the possible good stuff to merge.

I'll try to keep that in mind. What about putting that in the `_manual/pull_request.md` file?

When I'm at it, I saw that in the `_manual/group_permissions.md`, it is written the following:

>"Shown is an image of the permissions page for the Flyspray project's Contributors group"

but there is no image in there.

Two ways of dealing with that I guess:

1. add an image
2. remove the sentence

What do you think?

**Comment by peterdd** [comment]
> > Please do not put too much into one PR. One obstacle/problem and it delays the possible good stuff to merge.
> 
> I'll try to keep that in mind. What about putting that in the `manual/pull_request.md` file?

Sure, go ahead. :-) 

To ease reviewing and testing I prefer pull request that handle only one single problem.

And please give pull requests a good summary , '$name fixes' ..  well, I see which user does a pull request ;-)

File: _manual/translate.md [hunk 2]
```diff
@@ -7,13 +7,13 @@ order: 50
 
 <h3>Help Flyspray to get localized to your native language</h3>
 
-Want your language strings updated with the next Flyspray release? Pick your language file on https://github.com/Flyspray/flyspray/tree/master/lang , edit and make a pull request.
+Want your language strings updated with the next Flyspray release? Pick your language file on [the FlySpray github repository](https://github.com/Flyspray/flyspray/tree/master/lang), edit and make a pull request.
```
Comment (peterdd) [inline _manual/translate.md:10]: Writing Flyspray with the big S in the middle isn't used anymore. There are still some old translation files with that, but most updated languages and the 1.0 logo and website do not use that.

Comment (trebmuh) [reply _manual/translate.md:10]: ok

**Comment by trebmuh** [comment]
:+1:

**Pull request merged**